content/docs/reactivesearch/vue/search/SearchBox.md

```diff
@@ -1258,4 +1312,49 @@ The following events to the underlying `input` element:
 
 ## Examples
 
+### SearchBox with default props
+
 <a href="https://reactivesearch-vue-playground.netlify.com/?selectedKind=Search%20Components%2FSearchBox&selectedStory=Basic&full=0&addons=1&stories=1&panelRight=0" target="_blank">SearchBox with default props</a>
```

Comment:
@SavvyShah We are not using netlify anymore, please change the domain to use vercel.

-> https://reactivesearch-vue.vercel.app/